css/tablet.css

```diff
@@ -0,0 +1,163 @@
+/* Header Navbar */
+.menu {
+  position: absolute;
+  bottom: 12px;
+}
+
+.prof-logo {
+  display: none;
+}
```

Comment:
`<div class="d-none"></div>`

Reply:
if I add the class in the HTML then it will remove it from the phone resolution and it is easier to put `display: none` in tablet.css than adding `class="d-none"` and then in style.css add `display: block` and then again` display: none` in tablet.css